### PR TITLE
Added ansible helper file

### PR DESCRIPTION
**Pull request by asa1997**

### CODE REVIEW

File: src/main/bash/besman-ansible-helpers.sh [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+function __besman_run_ansible_playbook_extra_vars
+{
+    local playbook vars
+    playbook=$1
+    [[ ! -f $playbook ]] && __besman_echo_red "$playbook not found" && unset playbook vars && return 1
+    vars=$2
+    __besman_echo_white "Running $playbook with --extra-vars $vars"
+    ansible-playbook $playbook --ask-become-pass --extra-vars "$vars"
+    [[ "$?" ne 0 ]] && return 1
```
Comment (Pranshu021) [inline src/main/bash/besman-ansible-helpers.sh:90]: Check Relational error typo "-ne"

Comment (asa1997) [reply src/main/bash/besman-ansible-helpers.sh]: Resolved

**Comment by Pranshu021** [comment]
LGTM and Tested @sumodgeorge @panickervinod @sumodgeorge

**Review by Pranshu021 — APPROVED** [review]
LGTM and tested

**Pull request merged**